### DISCUSSION TITLE
Can't get a successful installation of windows 10 on amd64 on M1 machine

### DISCUSSION THREAD

**Discussion by Belg0** [utmapp/UTM#4122] [General]
**Describe the issue**
I'm following these instructions https://mac.getutm.app/gallery/windows-10 including the rosetta installation. I can get the machine to start, windows is installing. At the end of the installation if freezes**

**Configuration**
* UTM Version: Version 3.2.4 (58)
* OS Version: 12.4 (21F79)
* Intel or Apple Silicon? M1

**Crash log**
No crash only a never ever finishing installation of Windows 10 for:
 19041 x86
 19041 x64
 21390 x86
 21390 x64 

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by conath** [comment]
Please take a look at this video, as it shows how to install Windows 10 in UTM on M1. It is a different process than shown in the UTM gallery, and the video is showing older UTM. The process still applies. https://www.youtube.com/watch?v=KsNoGPczavw

**Comment by Belg0** [comment]
I got it to work, thanks!
